src/shared/XLinkDispatcherImpl.c

```diff
@@ -49,6 +55,10 @@ int dispatcherEventSend(xLinkEvent_t *event)
     mvLog(MVLOG_DEBUG, "Send event: %s, size %d, streamId %ld.\n",
         TypeToStr(event->header.type), event->header.size, event->header.streamId);
 
+    struct timespec stime;
+    clock_gettime(CLOCK_MONOTONIC, &stime);
```

Comment:
Lets modify this to be C++'s `std::chrono::steady_clock::now()` instead (check core for conversion to sec & nsec).
Can be added to some utils cpp file, and just a extern C function created for it.

https://www.man7.org/linux/man-pages/man3/clock_gettime.3.html as otherwise correct impl is CLOCK_MONOTONIC_RAW (but is non portable)